index.bs

```diff
@@ -883,6 +883,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     A [=[WAA]=] could be a [=roaming authenticator=], a dedicated hardware subsystem integrated into the [=client device=],
     or a software component of the [=client=] or [=client device=].
 
+    In general, an [=authenticator=] is assumed to have only one user.
+    If multiple natural persons share access to an [=authenticator=],
+    they are considered to represent the same user in the context of that [=authenticator=].
+    If an [=authenticator=] implementation supports multiple users in separated compartments,
+    then each compartment is considered a separate [=authenticator=]
+    with a single user with no access to other users' [=credentials=].
```

Comment:
perhaps "single" should be "given" ?

Reply:
I think "single" better emphasises that there's only one user per semantic authenticator.

Reply:
ok